experiment/references.md

```diff
@@ -1 +1,2 @@
-### Link your references in here
\ No newline at end of file
+1. Alex A. Granovsky, Firefly version 7.1.G, at [http://classic.chem.msu.su/gran/firefly/index.ph](http://classic.chem.msu.su/gran/firefly/index.php)
```

Comment:
Please use https links exclusively. Also give a meaningful link description instead of just pasting the link.